### ISSUE TITLE
 Almost all training sets ' NIR channel is  255 and the test set hanyang2002's NIR channel is 16bit.

### ISSUE THREAD

**Issue by wenhwu** [rulixiang/HanyangDataset#2]
Thank you for your generous sharing！

What's strange to me about the dataset is these two things:
- Almost all training sets ' NIR channel is  255.
- I have inquired the dynamic range of IKONOS is 11 bits per pixel. However,  the `hanyang2002`'s NIR channel is 16-bit, while the `hanyang2009`'s NIR channel is 11-bit.

The detailed information in [MtS-WH data set: color correction when extracting training set
](https://github.com/wenhwu/awesome-remote-sensing-change-detection/issues/3)
Is this normal? 
thanks
